### ISSUE TITLE
Position values not rounding in very low range (under 1.4 meters)

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#118]
While testing @mbarlow12's awesome work in #115 I noticed that
- when I jumped left with Mac's Function-key + Left arrow (same as Home key shortcut on Windows) I got a very long position value. The value read out to many decimal places.

I verified that position values under 1.4 (perhaps 1.3 or 1.2) meter mark are not rounding properly. The meter mark values in this low range are read out to about 18 decimal places.

**Here's a screen shot of the issue**
<img width="1018" alt="issue-rounding" src="https://user-images.githubusercontent.com/1902116/48187027-24371c80-e33a-11e8-9df2-822e1d46767f.png">

**Comment by terracoda** [comment]
@mbarlow12 this bug also exists in Gravity Force Lab Basics.

**Comment by jessegreenberg** [comment]
Just took a look since it came up in resistance-in-a-wire. Verified in the a11y view for gravity-force-lab
![capture](https://user-images.githubusercontent.com/6396244/48373759-c46da800-e690-11e8-9485-eee76df40ed1.PNG)

I actually wasn't able to reproduce in gravity-force-lab-basics though.

**Comment by terracoda** [comment]
@jessegreenberg, I thought I produced it in GFL:B, and I must be mistaken. In GLF:B the Mass 1 position slider actually goes to zero which is not correct. I do not think there should be any zero values. I will create a new issue for that in GFL:B repo for that issue.

**Comment by terracoda** [comment]
@mbarlow12, please see issue #119 before trying to fix this issue. #119 might resolve this issue.

**Comment by terracoda** [comment]
Awesome @mbarlow12, I'll close this issue and you can report about the new range in #119.
